### DISCUSSION TITLE
why every step (tick) that updates observables in an asynchronous process should be marked as action

### DISCUSSION THREAD

**Discussion by rluvaton** [mobxjs/mobx#3518] [Q&A] [answered]
Why it stated that:
> every step (tick) that updates observables in an asynchronous process should be marked as action

From [Mobx Docs | Asynchronous actions](https://mobx.js.org/actions.html#asynchronous-actions)https://github.com/mobxjs/mobx/blob/78d1aa2362b4dc5d521518688d6ac7e2d4f7ad3a/docs/actions.md?plain=1#L252

**Comment by urugator** [comment, marked as answer]
Maybe can help: https://github.com/mobxjs/mobx/discussions/3171#discussioncomment-1554807

**Reply by rluvaton** [reply]
thanks!
